containers/Dockerfile.buildenv

```diff
@@ -232,20 +209,16 @@ RUN wget http://github.com/xtensor-stack/xsimd/archive/refs/tags/8.1.0.tar.gz \
     && cmake -D CMAKE_BUILD_TYPE=Release -D BUILD_TESTS=OFF \
              -D CMAKE_INSTALL_PREFIX=/usr/local ../ \
     && make install \
-    && cd /work \
+    && cd ../.. \
```

Comment:
Why this change?

Reply:
For some reason, it would error if I left it as `cd /work`. Didn't dig into why